Tasks/VsTestV2/inputparser.ts

```diff
@@ -166,6 +166,14 @@ function getTestReportingSettings(inputDataContract : idc.InputDataContract) : i
     inputDataContract.TestReportingSettings.TestRunTitle = tl.getInput('testRunTitle');
     inputDataContract.TestReportingSettings.TestRunSystem = 'VSTS - vstest';
 
+    const resultsDir = path.resolve(tl.getVariable('System.DefaultWorkingDirectory'), tl.getInput('resultsFolder').trim());
+    if (utils.Helper.pathExistsAsDirectory(resultsDir.trim()))
+    {
+        inputDataContract.TestReportingSettings.TestResultsDirectory =  resultsDir;
+        tl.debug("TestResultsFolder: " + resultsDir);
+        ci.publishEvent({ 'TestResultsFolderUi': resultsDir } );
```

Comment:
temp or more than temp path
common.test or
defaultdir 

push this property of the path as you can't directly push path

Reply:
Added. Is there a better way to add this?